src/pathfinding_service/cli.py

```diff
@@ -81,9 +90,19 @@ def main(  # pylint: disable-msg=too-many-arguments
     info_message: str,
     enable_debug: bool,
     matrix_server: List[str],
+    accept_disclaimer: bool,
 ) -> int:
     """ The Pathfinding service for the Raiden Network. """
     log.info("Starting Raiden Pathfinding Service")
+    click.secho(
+        DISCLAIMER, fg="yellow",
+    )
+    if not accept_disclaimer:
+        click.confirm(
+            "\nHave you read, understood and hereby accept the above "
```

Comment:
You could also make the message a constant and use it everywhere

Reply:
indeed, done